AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -23,6 +25,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, assign) BOOL isMutable;
 
+@property (nonatomic, strong, nullable) ASTraitCollection *traitCollection;
```

Comment:
@rcancro this is probably the main one I'd change usage of.  This could potentially be a create-on-demand property but should probably be readonly.  The writing path could use environment trait collection, which since this already conforms to ASEnvironment, it at least has the struct.